src/dispatch/plugins/dispatch_slack/endpoints.py

```diff
@@ -46,10 +60,11 @@ def get_request_handler(request: Request, body: bytes, organization: str) -> Sla
     )
     for p in plugin_instances:
         if is_current_configuration(body=body, headers=request.headers, plugin_instance=p):
-            incident_configure(p.configuration)
-            workflow_configure(p.configuration)
             case_configure(p.configuration)
             feedback_configure(p.configuration)
+            incident_configure(p.configuration)
+            workflow_configure(p.configuration)
+            app._configuration = p.configuration
```

Comment:
Can we just put `app._configuration` in `context_properties` dictionary and remove `configuration_middleware()`?